dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
> I think we're only doing absolute redirects and might not need RewriteBase at all, will test.

We *definitely* need RewriteBase because the dlang.org server does not have a sensible RewriteBase for some reason.